cmd/podman/main_local.go

```diff
@@ -69,7 +69,7 @@ func init() {
 	rootCmd.PersistentFlags().StringArrayVar(&MainGlobalOpts.StorageOpts, "storage-opt", []string{}, "Used to pass an option to the storage driver")
 	rootCmd.PersistentFlags().BoolVar(&MainGlobalOpts.Syslog, "syslog", false, "Output logging information to syslog as well as the console")
 
-	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.TmpDir, "tmpdir", "", "Path to the tmp directory")
+	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.TmpDir, "tmpdir", "", "Path to the tmp directory for libpod state content.\n\nNote: use environment variable 'TMPDIR' to change temporary storage location for container images, '/var/tmp'.\n")
```

Comment:
suggest add 'the' to  "the environment" and "the temporary storage"

Reply:
fixed